src/main/java/it/gov/pagopa/pu/pagopapayments/util/ConversionUtils.java

```diff
@@ -58,4 +58,9 @@ public static OffsetDateTime localDate2RomeMaxTime(LocalDate dueDate){
       .orElse(MAX_EXPIRATION_DATE);
   }
 
+  public static LocalDateTime localDateToLocalDateTime(LocalDate localDate) {
+    return Optional.ofNullable(localDate)
+      .map(dt -> dt.atTime(LocalTime.MAX).atZone(ConversionUtils.ZONE_ID_ROME).toLocalDateTime())
+      .orElse(null);
+  }
```

Comment:
it doesn't need a common method just to convert a LocalDate into a LocalDateTime
`localDate.atTime(LocalTime.of(...))` (neither the provided impl is the right way)

It could be useful to a have a common method to transform it into a LocalDateTime at a particular time.
It already contains a utility method to transform it into StartOfDay, you could introduce here a atEndOfDay `method`